components/function-controller/README.md

```diff
@@ -119,16 +117,7 @@ To deploy the Function Controller to the `serverless-system` Namespace, run:
 make deploy
 ```
 
-This runs [ko](https://github.com/google/ko) to build your image and push it to your configured docker repository. 
-
->**NOTE:** Make sure the  **KO_DOCKER_REPO**  environment variable points to a valid docker repository.
->```bash
->#for hub.docker.io
->export KO_DOCKER_REPO=<your dockerhub username>
->
->#to use your local docker daemon and its image store
->export KO_DOCKER_REPO=ko.local
->```
+This runs [ko](https://github.com/google/ko) to build your image and push it to the container repository set in the **KO_DOCKER_REPO** environment variable.
```

Comment:
How about using this link https://github.com/google/ko#usage?

Reply:
This was not added as part of this PR, but from what I see it's just a link to the project. I think it's useful to start reading what the project is before reading how to use the project. Besides, you don't need knowledge about the usage since the user runs a `make` command, no `ko` directly.
Will change if you feel strong about it.

Reply:
My rationale was: since we removed how to set env variables, why not guide the user to the exact location of how to use ko. Sure, you may look into the whole documentation(by scrolling upwards). But this is about usage so I think lets to get to the part which will let me use it rather than scroll down or read about ko until I find how to use it. You may keep it if you like to.